guide/additional-info/changes-in-v14.md

```diff
@@ -520,24 +526,37 @@ Added `Guild#maxVideoChannelUsers` as of 14.2.0 which indicates the maximum numb
 
 `videoQualityMode` may be used whilst creating a channel to initially set the camera video quality mode.
 
+### GuildEmojiManager
+
+Added `GuildEmojiManager#delete()` and `GuildEmojiManager#edit()` for managing existing guild emojis.
+
+### GuildForumThreadManager 
+
+Added `GuildForumThreadManager` as manager for threads in forum channels as of 14.4.0.
+
 ### GuildMemberManager
 
 Added `GuildMemberManager#fetchMe()` to fetch the client user in the guild.
-
 Added `GuildMemberManager#addRole()` and `GuildMemberManager#removeRole()` as of 14.3.0. These methods allow a single addition or removal of a role respectively to a guild member, even if uncached.
 
-### GuildEmojiManager
+### GuildTextThreadManager
 
-Added `GuildEmojiManager#delete()` and `GuildEmojiManager#edit()` for managing existing guild emojis.
+Added `GuildTextThreadManager` as manager for threads in text channels and announcement channels as of 14.4.0.
 
 ### Interaction
 
 Added `Interaction#isRepliable()` to check whether a given interaction can be replied to.
 
+### Message
+
+`Message#position` has been added as an approximate position in a thread as of 14.4.0.
+
 ### MessageReaction
 
 Added `MessageReaction#react()` to make the client user react with the reaction the class belongs to.
 
 ### Webhook
 
 Added `Webhook#applicationId`.
+
+Added `Webhook#threadName` as of 14.4.0 which allows a webhook to create a post in a forum channel.
```

Comment:
```suggestion
Added `threadName` property in `Webhook#send()` options as of 14.4.0 which allows a webhook to create a post in a forum channel.
```
Is it good?

Reply:
@almostSouji ?

Reply:
Added property X in Y options...

Reply:
I can't find anything more creative

Reply:
Unsure what you mean by creative? use what you suggested and apply the order changes I requested above and it's fine